docs/community/zkillboard/index.md

```diff
@@ -20,3 +20,14 @@ maintainer:
 
 </div>
 
+## About
+
+<img src="https://github.com/user-attachments/assets/63bdc864-b2f0-4d8e-b6ce-15ba2525ce19" style="width: 64px;" />
```

Comment:
All existing project pages upload the icon along with the page, I think that's better to avoid requiring a request to another website (and future link rot).

Reply:
Another website?  The image is hosted by github.

Reply:
Image has been removed.